k4MarlinWrapper/k4MarlinWrapper/converters/EDM4hep2Lcio.h

```diff
@@ -64,6 +67,9 @@ class EDM4hep2LcioTool : public AlgTool, virtual public IEDMConverter {
 
   PodioDataSvc*                   m_podioDataSvc;
```

Comment:
Could we have a one line description for each service under what circumstances which service is used?

Reply:
Like the ones I added?